### PR TITLE
creation of blogs and images for the 3.18 release

### PR DESCRIPTION
**Pull request by LaureenH**

### CODE REVIEW

File: blogposts/2020/3.18/C++-precise-code.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
+---
+title: 'New C++ precise code intelligence solution'
+author: 'Garo Brik'
+publishDate: 2020-07-20T00:00-07:00
+tags: ["blog"]
+slug: indexed-non-master-branches
+
+---
+## New C++ precise code intelligence solution (#[10175](https://github.com/sourcegraph/sourcegraph/issues/10175))
```
Comment (christinaforney) [inline blogposts/2020/3.18/C++-precise-code.md:10]: Should this be a H1 (i.e. #)?

File: blogposts/2020/3.18/C++-precise-code.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (christinaforney) [inline blogposts/2020/3.18/C++-precise-code.md:1]: Extra newline

File: blogposts/2020/3.18/C++-precise-code.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+---
+title: 'New C++ precise code intelligence solution'
+author: 'Garo Brik'
+publishDate: 2020-07-20T00:00-07:00
+tags: ["blog"]
+slug: indexed-non-master-branches
+
+---
```
Comment (christinaforney) [inline blogposts/2020/3.18/C++-precise-code.md:9]: I think you're missing two metadata items here:
```
heroImage: https://about.sourcegraph.com/sourcegraph-mark.png
published: true
```

File: blogposts/2020/3.18/indexed-non-master-branches.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+
```
Comment (christinaforney) [inline blogposts/2020/3.18/indexed-non-master-branches.md:1]: Same feedback on this file as in the first one.

File: blogposts/2020/3.18/search-across-revisions.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+![search over Sourcegraph versions 3.16 and 3.17](images/searchacrossrev1.png)
+
+
+A search over all branches, using [@*refs/heads/](https://sourcegraph.com/search?q=repo:%5Egithub.com/sourcegraph/sourcegraph%24%40*refs/heads/+CONTAINER_ID&patternType=literal&case=yes) would look like this
+
+![search over all branches using @*refs/heads/](images/searchacrossrev2.png)
+
+
+Be aware that searching over all branches may be slow. The cost of searching a branch is the same cost as searching a repository. To speed this up, ensure that branches are indexed with our experimental [Indexed non-master branches” feature](add link).
```
Comment (christinaforney) [inline blogposts/2020/3.18/search-across-revisions.md:28]: Is the link still WIP?

File: blogposts/2020/3.18/indexed-non-master-branches.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+[HEAD](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefHEADaHEAD) is a [symbolic reference](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefsymrefasymref) to the current branch in git. The HEAD of a repository you clone is your default branch (by default “master” in Git). So “HEAD” in Sourcegraph is your default branch.
+
+
+### Example configuration
+
+Branches:
+
+![alt_text](images/expfeatures1.png "image_tooltip")
```
Comment (christinaforney) [inline blogposts/2020/3.18/indexed-non-master-branches.md:24]: Should this have proper alt text and tooltip text? Same with image below.

File: blogposts/2020/3.18/C++-precise-code.md (hunk 3 above)
Comment (LaureenH) [reply blogposts/2020/3.18/C++-precise-code.md:9]: <3

**Pull request merged**